src/api/routes/company.js

```diff
@@ -174,4 +176,30 @@ export default (app) => {
                 return next(err);
             }
         });
+
+    /**
+     * Verifies if a company has reached max concurrent offers between two dates
+     */
+    router.get("/:companyId/hasReachedMaxConcurrentOffersBetweenDates",
+        validators.setDefaultValuesConcurrent,
```

Comment:
If validation only happens after this line, there's no guarantee you'll receive valid dates, which will make this function crash (#191)